docs/getting-started/installation.md

```diff
@@ -32,7 +32,8 @@ The server may need a few seconds to start up so be patient and wait for the cor
     3965 [gremlin-server-boss-1] INFO  org.apache.tinkerpop.gremlin.server.GremlinServer  - Channel started at port 8182.
     ```
 
-We can now start a Gremlin Console on our local device and try to connect to the new server:
+We can now start a Gremlin Console and try to connect to the new server. We can do so either on our local device (see local
+installation section below) or inside the Docker container.
```

Comment:
_inside the Docker container_ <- Do we really want to document that? I think it's easier and a best practice in general with Docker containers to not start a second process inside the Docker container, but to instead start a different container for the console.

Reply:
I understand the confusion and it probably also originated from this section having been below the local installation originally.
But my problem with this approach is that it's recommending something that is generally considered to not be good practice for Docker. See for example [this part of the official Docker docs](https://docs.docker.com/config/containers/multi-service_container/):

> It is generally recommended that you separate areas of concern by using one service per container.

or [this tutorial](https://www.tutorialworks.com/containers-single-or-multiple-processes/): 

> One of these principles is that there should just be one process running in a container. That is to say, a Docker container should have just one program running inside it.

How about if we avoid the confusion by only linking to the local installation section for the Gremlin Console here and then directly continuing with the approach that starts a second JanusGraph container for the console?

Or if you really want to document starting the console within the same container, then I would mention that this is only meant for quick testing or something like that.

Reply:
> But my problem with this approach is that it's recommending something that is generally considered to not be good practice for Docker

Well said. Let me try to paraphrase this section and see if we can make it more clear to users.